tests/rbd/test_rbd_image_migration_qcow.py

```diff
@@ -0,0 +1,191 @@
+"""Module to execute Live image migration with qcow data format.
+
+Pre-requisites :
+- Cluster configured with atleast one client node with ceph-common package,
+conf and keyring files
+- access qcow2 data from http://download.ceph.com/qa/ubuntu-12.04.qcow2
+
+Test cases covered :
+CEPH-83584070 - Live migration of images with external data source as QCOW2 data format
+
+Test Case Flow :
+1. Create two pools with rbd application enabled
```

Comment:
If this is a scale level implementation then 2 pools is not true right? 
I see from test suite that you are executing it at scale. So please change the test case steps in polarion and here accordingly.

Reply:
Done.